package.json

```diff
@@ -40,10 +43,17 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "@types/argparse": "^1.0.38",
```

Comment:
Types don't exist at runtime so it might be useful to move them to the devDependencies section of package.json.

Reply:
Yep, still need to do some cleanup on the package.json.

Reply:
Fixed in 9262563